mobile/src/screens/note/libs/constants.ts

```diff
@@ -0,0 +1,21 @@
+const DEFAULT_NOTE_VALUE = {
```

Comment:
Duplicate

Reply:
it is empty strings of values when user firstly writes smth to note, because i was said to have placeholders, but i need to have default empty value

Reply:
Please checkout with frontend part